core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -1443,5 +1446,32 @@ public static int getDictionaryChunkSize() {
     }
     return dictionaryOneChunkSize;
   }
+
+  /**
+   * @param csvFilePath
+   * @return
+   */
+  public static String readHeader(String csvFilePath) {
+
+    DataInputStream fileReader = null;
+    BufferedReader bufferedReader = null;
+    String readLine = null;
+
+    try {
+      fileReader =
+          FileFactory.getDataInputStream(csvFilePath, FileFactory.getFileType(csvFilePath));
+      bufferedReader =
+          new BufferedReader(new InputStreamReader(fileReader, Charset.defaultCharset()));
```

Comment:
@foryou2030 ....instead of using Charset.defaultCharset() use the below line of code.
Charset.forName( CarbonCommonConstants.DEFAULT_CHARSET)